tests/streaming.rs

```diff
@@ -185,32 +183,6 @@ fn it_processes_usage_event_across_chunks_until_done() {
             Some(LogLevel::Debug),
             Some("#2 handle_stream: processing chunk: data: [DONE]\n\n"),
         )
-        .execute_and_expect(ReturnType::Action(Action::Pause))
-        .unwrap();
-
-    // Third call: end_of_stream true → should now send gRPC using captured usage
```

Comment:
Why removing this? Perhaps related to `wasm log: proxy_on_grpc_receive invalid context_id, no match found     thread=36` ?